core/src/test/java/org/apache/accumulo/core/crypto/CryptoTest.java

```diff
@@ -289,10 +319,20 @@ public void testRFileEncrypted() throws Exception {
       }
     }
 
-    Scanner iter =
-        RFile.newScanner().from(file).withFileSystem(fs).withTableProperties(cryptoOnConf).build();
+    // test to make sure the RFile is encrypted
     ArrayList<Key> keysRead = new ArrayList<>();
-    iter.forEach(e -> keysRead.add(e.getKey()));
+    try (Scanner iter = RFile.newScanner().from(file).withFileSystem(fs).build()) {
+      iter.forEach(e -> keysRead.add(e.getKey()));
+      fail("File was not encrypted.");
+    } catch (UncheckedIOException e) {
```

Comment:
```suggestion
      assertThrows(UncheckedIOException.class, () -> iter.forEach(e -> keysRead.add(e.getKey())),
          "The file was expected to be encrypted but was not");
```
This might be a better place for `assertThrows` than try-catch

Reply:
Yep, thanks!